Microsoft.NET.Build.Containers/ContainerHelpers.cs

```diff
@@ -9,6 +9,16 @@ public static class ContainerHelpers
 
     private static Regex imageNameRegex = new Regex("^[a-z0-9]+([._-][a-z0-9]+)*(/[a-z0-9]+([._-][a-z0-9]+)*)*$");
 
+    /// <summary>
+    /// Matches if the character is not lowercase or numeric
+    /// </summary>
+    private static Regex imageFirstNameCharacters = new Regex("[^a-z0-9]");
```

Comment:
Looks like this isn't used

Reply:
good spot - this was turned into a more explicit character check